.github/workflows/reusable-build-test-release.yml

```diff
@@ -783,6 +789,39 @@ jobs:
           path: |
             *_markdown.txt
 
+  appinspect-api:
+    name: appinspect api ${{ matrix.tags }}
+    needs: build
+    if: |
+      always() &&
+      needs.build.result == 'success'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        tags:
+          - "cloud"
```

Comment:
What is the reasoning behind for choosing those 3 tags?

Reply:
This is what we did during QAC